src/utils/getProfileData.ts

```diff
@@ -54,12 +54,30 @@ function requestAccessToken({origin}: URL): Promise<SubjectAccessToken> {
 function formatLiquidProfileData(
   entries: ProfileNode[],
 ): FormattedProfileNode[] {
-  return entries.map(function(entry: ProfileNode) {
-    if (!entry.partial.includes(':')) {
-      entry.partial = `snippet:${entry.partial}`;
+  return entries.map((entry: ProfileNode) => {
+    const nameParts = entry.partial.split('/');
+    let name = '';
+    let filename = null;
+    if (nameParts.length === 1 && !entry.partial.includes(':')) {
+      name = `snippet:${entry.partial}`;
+      filename = `snippets/${entry.partial}.liquid`;
+    } else if (/shopify:\/\/apps/.test(entry.partial)) {
+      name = `app-${nameParts[4].slice(0, -1)}:${nameParts[3]}`;
```

Comment:
Instead of having a hardcoded `app-<section|snippet>:<app-title>` I think it would be more helpful to users if we highlighted the extension's handle. Maybe it could be something like

So the generic form would be `app-<section-snippet>:<handle>:<app-title>` and a concrete example would be `app-snippet:star-rating:product-reviews`